src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -159,6 +159,293 @@
             --enable-compression
 """
 
+helps['cdn endpoint rule'] = """
+type: group
+short-summary: Manage delivery rules for an endpoint.
+"""
+
+helps['cdn endpoint rule add'] = """
+type: command
+short-summary: Add a delivery rule to a CDN endpoint.
+parameters:
+  - name: --rule-name
+    type: string
+    short-summary: Name of the rule.
+  - name: --order
+    type: string
+    short-summary: Order of the rule.
```

Comment:
I'm not familiar with it. `--order` means `--order first`? or `--order 10000`? Is there any constrain for this argument?